Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/ConfigChangeDetector.kt

```diff
@@ -60,4 +64,19 @@ class ConfigChangeDetector @Inject constructor(
             RiskLevelRepository.setRiskLevelScore(RiskLevel.UNDETERMINED)
         }
     }
+
+    @VisibleForTesting
+    internal suspend fun checkForDiagnosisKeysDataMapping(exposureWindowRiskCalculationConfig: ExposureWindowRiskCalculationConfig) {
```

Comment:
Is this also invoked if the app config cannot be loaded initially and the client falls back to the default app config?

Reply:
config change detector is no longer used